### DISCUSSION TITLE
Deprecate (or use?) react-location npm package

### DISCUSSION THREAD

**Discussion by enisdenjo** [TanStack/router#245] [Ideas]
I mistakenly installed and used the [`react-location` npm package](http://npm.im/react-location) (assumed it's the right name following the name of the repo).

However, I've only noticed today that the actual name is prefixed with `@tanstack/` and that I've been using a 2 month old version. 😅

**Comment by tannerlinsley** [comment]
I can add some logs to deprecate it :)
